### PR TITLE
fix: cleanupJobResource

### PR DESCRIPTION
**Pull request by ANGkeith**
`docker rm` and `docker volume rm` supports multiple args. 

this should probably be slightly more efficient.

Not too sure why, but the current cleanup also seemed to leave some `docker volume` undeleted

To replicated the issue:

```yaml
# .gitlab-ci.yml
job0:
  image: bash
  script: echo hi
```

```bash
for i in $(seq 1 9); do
 gitlab-ci-local
done
```

Run `docker volume ls` and observe that there are remaing `xxxxx-tmp` volume that are not cleaned up

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -41,7 +41,8 @@ process.on("SIGINT", async (_: string, code: number) => {
                     } else {
                         process.stderr.write(chalk`{red ${e.stack ?? e}}\n`);
                     }
-                    cleanupJobResources(jobs).finally(process.exit(1));
```
Comment (ANGkeith) [inline src/index.ts:44]: with this,

process seemed to exit before cleanup, any idea why ? 


Run this, 
```yaml
cache:
  key:
    files:
      - /foo
  paths:
    - ${CI_PROJECT_DIR}/node_modules
job0:
  image: bash
  script: echo hi
```

and then `docker ps -a`

Comment (ANGkeith) [reply src/index.ts:44]: Using the following .gitlab-ci.yml
```yaml
cache:
  key:
    files:
      - /foo
  paths:
    - ${CI_PROJECT_DIR}/node_modules
job0:
  image: bash
  script: echo hi
 ```
 
 and running `gitlab-ci-local`
 
 the cleanup works for the `await` syntax but not `finally` syntax

Comment (firecow) [reply src/index.ts:44]: await is a blocking statement

cleanupJobResources(jobs).finally(process.exit(1)); doesn't block anything

Comment (firecow) [reply src/index.ts:44]: I imagine the program shuts down, before even getting to the callback function in finally.
So it kindda makes sense.

**Review by firecow — APPROVED** [review]

**Pull request merged**